src/main/resources/data/products.json

```diff
@@ -47,5 +47,19 @@
     "productKindName": "Investment Account",
     "productTypeId": "7777777",
     "productTypeName": "Investment Account"
+  },
+  {
+    "id": "8",
+    "productKindId": "kind8",
+    "productKindName": "Pocket Parent",
+    "productTypeId": "8888888",
+    "productTypeName": "Pocket Parent"
+  },
+  {
+    "id": "9",
+    "productKindId": "kind9",
+    "productKindName": "Pocket",
+    "productTypeId": "9999999",
+    "productTypeName": "Pocket"
```

Comment:
Are these product kinds already configured within product summary? Because otherwise they might be categorised under "Custom Product Kinds". Not sure if that's an issue for pockets.

Reply:
@lehphyro Do you know if these product kinds are already configured within product summary?

Reply:
You know that because you use them to create pockets :)
Here is a PR created by yourself: https://stash.backbase.com/projects/PFM/repos/pocket-management/pull-requests/14/diff#pocket-tailor/src/main/resources/application.yml

Reply:
Annatolli Volkodav provided me this info [https://stash.backbase.com/projects/AR/repos/arrangement-manager/pull-requests/393/diff#persistence/src/main/resources/sql/mssql/create/arrangement-manager.sql?t=128] , which seems to correspond with the settings in the products.json file.